Formula/qt5.rb

```diff
@@ -86,6 +87,7 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :postgresql => :optional
   depends_on :xcode => :build
+  depends_on "openssl" => :optional
```

Comment:
```
depends_on "openssl" if build.with?("postgresql") || build.with?("mysql")
```
is probably right since if openssl is a non-default option as it is in the PR right now, there will be a lot of people building qt5 from source for a rather – let's say "confused" – reason.